charts/jenkins.io/templates/zh-configmap.yaml

```
@@ -16,10 +16,21 @@ data:
       }
     {{- end }}
   
+      root   /usr/share/nginx/html;
+
       location / {
-          root   /usr/share/nginx/html;
           index  index.html index.htm;
       }
+
+      location ~* \.(js|css|png|jpg|jpeg|gif|ico)$ {
```

Add the fonts in too.
Eventually add cors to the static assets too so plugin sites can use it instead of copying their own

https://coderwall.com/p/ad9ifw/nginx-aggressive-asset-configuration

```suggestion
      location ~* \.(?:css|js|woff|eot|svg|ttf|otf|png|gif|jpe?g) {
```